domain/consensus/processes/consensusstatemanager/find_selected_parent_chain_changes.go

```diff
@@ -0,0 +1,54 @@
+package consensusstatemanager
+
+import "github.com/kaspanet/kaspad/domain/consensus/model/externalapi"
+
+func (csm *consensusStateManager) findSelectedParentChainChanges(
```

Comment:
Can you rename to selectedParentChainChanges or calculateSelectedParentChainChanges